### PR TITLE
CLN: event-model recipe cleanup

### PR DESCRIPTION
**Pull request by mrakitin**
Here are the improvements:
- 2 blank spaces indent instead of 4
- update the maintainers list
- `pip install ...` instead of `python setup.py ...`

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mrakitin** [comment]
@conda-forge-admin, please rerender

**Comment by mrakitin** [comment]
@CJ-Wright, ready for review.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,42 +1,39 @@
...
-    build:
-        - python >=3.6
-        - setuptools
-
-    run:
-        - python >=3.6
-        - setuptools
-        - jsonschema
-        - numpy
+  build:
```
Comment (CJ-Wright) [inline recipe/meta.yaml]:
```suggestion
  host:
```

**Comment by CJ-Wright** [comment]
LGTM thank you very much @mrakitin !

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,42 +1,39 @@
...
-    license: BSD 3-Clause
-    summary: data model for event-based data collection and analysis
+  home: https://github.com/NSLS-II/event-model
+  license: BSD 3-Clause
+  summary: data model for event-based data collection and analysis
 
 extra:
-    recipe-maintainers:
-        - ericdill
-        - licode
```
Comment (CJ-Wright) [inline recipe/meta.yaml:40]: @licode @ericdill are you ok with being removed from this recipe?

**Pull request merged**